cypress/e2e/versions.spec.js

```diff
@@ -25,7 +25,7 @@ describe('Versions', () => {
 
 			cy.get('[data-cy-files-list-row-name="' + versionFileName + '"] [data-cy-files-list-row-mtime]').click()
 			cy.get('.app-sidebar-header').should('be.visible').should('contain', versionFileName)
-			cy.get('.app-sidebar-tabs__tab[data-id="version_vue"]').click()
+			cy.get('.app-sidebar-tabs__tab:contains("Versions")').click()
```

Comment:
Nice! Did not know about the `:contains` selector. That may come in handy at some point. :smile: